### PR TITLE
Scaffold page

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -135,13 +135,23 @@
...
+<?php
+    // or is people archive
+      } else if (is_post_type_archive('people')) {
 ?>
           <a href="<?php echo home_url('profile/'); ?>"><li>Profile</li></a>
           <a href="<?php echo home_url('news/'); ?>"><li>Clients</li></a>
           <a href="<?php echo home_url('awards/'); ?>"><li>Awards</li></a>
-          <a href="<?php echo home_url('people/'); ?>"><li>People</li></a>
+          <a href="<?php echo home_url('people/'); ?>"><li class="font-color-active">People</li></a>
           <a href="<?php echo home_url('recruit/'); ?>"><li>Recruit</li></a>
```
Comment (mixcas) [inline header.php:155]: why not be conditional only to the people li item? the rest of the list could be the same code, no?

Comment (varffs) [inline header.php:155]: just seemed odd to have the double conditional for the if and then further conditionals inside

Comment (varffs) [inline header.php:155]: also the $post global wont be defined if a post type archive i dont think. or maybe it is the first post not sure

Comment (mixcas) [inline header.php:155]: mmkay

**Pull request merged**